andela/ah-the-unsullied-frontend
# #162949164 Configure the redux store

#### What does this PR do?

Configure the redux store and file structure

#### Description of Task to be completed?
- [x] View redux store on browser dev tools

#### How should this be manually tested?

* Git clone this repo: `git clone https://github.com/andela/ah-the-unsullied-frontend.git`
* Cd to the cloned repo
* Git checkout to ch-redux-store-162949164 
* Make sure have the redux dev tool extension
* npm start :-This should open up a browser tab
* Click on the redux dev tools icon on the browser
* This should open up the empty redux store - make sure it looks like the screenshot below.

#### Any background context you want to provide?
None
#### What are the relevant pivotal tracker stories?
[#162949164](https://www.pivotaltracker.com/story/show/162949164)

#### Screenshots (if appropriate)
Redux store on dev tools:
<img width="1673" alt="redux" src="https://user-images.githubusercontent.com/33450849/52361425-492d4f80-2a4f-11e9-8ce5-2c05127df238.png">

#### Questions:
